client/src/locales/en-US.json

```diff
@@ -15,14 +15,14 @@
         "no": "Cancel"
     },
     "openFile": {
-        "failed": "Could not open the element at `{path}`."
+        "failed": "Failed to open element at `{path}`."
```

Comment:
`element` may be too abstract?
We should probably define what we a calling a file, a folder, and something that can be either (I think I used the word `entry` most of the time).